### PR TITLE
New Blog post about PyLadiesCon

### PR DESCRIPTION
**Pull request by georgically**
New Blog post about PyLadiesCon

### CODE REVIEW

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+description: "PyPodcats Partners with PyLadiesCon 2024"
+date: 2024-09-26T00:00:00Z
+categories: ["Blog Post",]
+author: "Georgi"
+tags: ["python", "community", "conference", "pyladiescon", "talks"]
+draft: false
+image: "/images/pyladiescon-2024.png"
+---
+
+PyLadiesCon is back, stronger than ever for its second year, and PyPodcats is thrilled to be an official media partner!
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:13]:
```suggestion
[PyLadiesCon](https://conference.pyladies.com) is back, stronger than ever for its second year, and PyPodcats is thrilled to be their official media partner! As the media partner, we are sharing our podcast-creation resources, like [StreamYard and Veed.io subscriptions](/about/financials), with PyLadiesCon and PyLadiesCon speakers.
```

**Review by terezaif — APPROVED** [review]
Lgtm!

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+At PyPodcats, we've always advocated for the underrepresented Pythonistas in our community, and we're proud to support PyLadiesCon in 2024.
+
+## What is PyLadiesCon?
+
+If you haven’t heard yet,[The PyLadies Conference (PyLadiesCon)](https://conference.pyladies.com/) is the place to be this December! It’s all happening online from December 6-8, 2024, and the best part? It’s FREE. This event is all about empowering PyLadies, spreading knowledge, and embracing diversity in the Python world.
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:19]:
```suggestion
(PyLadiesCon)(https://conference.pyladies.com/) is the global conference for the [PyLadies](https://pyladies.com) community, and it is **the place to be** this December! It’s all happening online from December 6-8, 2024, and the best part? It’s FREE. This event is all about empowering PyLadies, spreading knowledge, and embracing diversity in the Python world.
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+### Multilingual Talks and Sessions
+
+One of the coolest things about PyLadiesCon is that the speakers get to present in whichever language they’re most comfortable with. Yep, it’s a fully multilingual event—no need to stress over language barriers here! So attendees will be also able to watch the talks with captioning if the talk is not delivered in the language the attendee preferred.
+
+### What to Expect at PyLadiesCon
+
+Here’s a quick outline:
+##### - Talks : Expect tons of insightful talks covering everything from the latest Python trends to best practices and cutting-edge tech from our inspiring speakers.
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:28]:
```suggestion
##### Talks
Expect tons of insightful talks covering everything, from the latest Python trends to the best practices, and cutting-edge tech from our inspiring speakers. 
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+One of the coolest things about PyLadiesCon is that the speakers get to present in whichever language they’re most comfortable with. Yep, it’s a fully multilingual event—no need to stress over language barriers here! So attendees will be also able to watch the talks with captioning if the talk is not delivered in the language the attendee preferred.
+
+### What to Expect at PyLadiesCon
+
+Here’s a quick outline:
+##### - Talks : Expect tons of insightful talks covering everything from the latest Python trends to best practices and cutting-edge tech from our inspiring speakers. 
+
+{{<figure src="/images/thais.png" caption="Thais Viana's Keynote session at PyLadiesCon 2023" alt="Thais Viana's Keynote session at PyLadiesCon 2023">}}
+
+##### - Panel Discussions: Get ready for engaging sessions, from Python dev hacks to career advice and more.
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:32]:
```suggestion
##### Panel Discussions
Get ready for engaging sessions, from Python dev hacks to career advice, and more.
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+{{<figure src="/images/panel.png" caption="APAC panel discussion at PyLadiesCon 2023" alt="APAC panel discussion at PyLadiesCon 2023">}}
+
+And wait, there’ll be loads of networking!
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:36]:
```suggestion
##### Networking

Connect and get to know other PyLadies worldwide. Build your network and expand your horizons. Open yourself up to new opportunities.
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## What’s New for 2024?
+
+This year, over 190 talks were submitted—so you know the content is going to be top-notch! Talks will be livestreamed on the PyLadies YouTube channel, and you can hop into live Q&A sessions (if the speaker is up for it).
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:41]:
```suggestion
This year, [over 190 talks were submitted](https://conference.pyladies.com/news/cfp-close/) — so you know the content is going to be top-notch! Talks will be live-streamed on the PyLadies YouTube channel, and you can hop into live Q&A sessions.
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+## Fun Fact Time!
+
+Did you know some of our very own PyPodcats guests were speakers at PyLadiesCon 2023?
+
+#### [Joanna Jablonski](/speakers/joanna-jablonski/) 
+Our guest from [Hidden Figures of Python Episode 1](/episodes/ep-1/), and a recent recipient of the [PSF Community Service Award](https://www.python.org/community/awards/psf-awards/#october-2024:~:text=The%20Q3%20Community%20Service%20Award%20was%20given%20to%20Joanna%20Jablonski%20for%20her%20multifaceted%20contributions%20across%20various%20domains%2C%20including%20her%20work%20as%20contributor%20to%20CPython%2C%20an%20interpreter%20author%2C%20and%20an%20architect%20of%20developer%20education%20tools%20and%20tutorials), gave an awesome talk titled [How to Make a Docs Site: Shortcuts for Busy Devs](https://www.youtube.com/watch?v=N3mO4MEtpwA).
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:48]:
```suggestion
Our guest from [Episode 1](/episodes/ep-1/), and a recent recipient of the [PSF Community Service Award](https://www.python.org/community/awards/psf-awards/#october-2024:~:text=The%20Q3%20Community%20Service%20Award%20was%20given%20to%20Joanna%20Jablonski%20for%20her%20multifaceted%20contributions%20across%20various%20domains%2C%20including%20her%20work%20as%20contributor%20to%20CPython%2C%20an%20interpreter%20author%2C%20and%20an%20architect%20of%20developer%20education%20tools%20and%20tutorials), gave an awesome talk titled [How to Make a Docs Site: Shortcuts for Busy Devs](https://www.youtube.com/watch?v=N3mO4MEtpwA).
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+
+#### [Mona Obedoza](/speakers/mona-obedoza/)
+Featured in [Hidden Figures of Python Episode 5](/episodes/ep-5/), Mona gave a talk about her project, [TEDDY (Text-based Early Distress Detector for Youth)](https://www.youtube.com/watch?v=DB7xxXHH-mo).
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:52]:
```suggestion
Featured in [Episode 5](/episodes/ep-5/), Mona gave a talk about her project, [TEDDY (Text-based Early Distress Detector for Youth)](https://www.youtube.com/watch?v=DB7xxXHH-mo).
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## Wanna Get Involved?
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:55]:
```suggestion
## Get Involved
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+PyLadiesCon is still looking for volunteers! If you want to help out, there are plenty of roles to fill, including:
+
+Communications Committee
+Design & Social Media
+Sponsorship & Finance
+Tech & Infrastructure
+Programs Committee
+Website Committee
+Curious? Check out the [volunteer page](https://conference.pyladies.com/volunteer/) and get in on the action!
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:65]:
```suggestion
- Communications Committee
- Design & Social Media
- Sponsorship & Finance
- Tech & Infrastructure
- Programs Committee
- Website Committee

Check out the [volunteer page](https://conference.pyladies.com/volunteer/) and get in on the action!


```

**Review by Mariatta — COMMENTED** [review]
Just some suggestions about copy editing. Thank you! I love this.

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+image: "/images/pyladiescon-2024.png"
+---
+
+[PyLadiesCon](https://conference.pyladies.com) is back, stronger than ever for its second year, and PyPodcats is thrilled to be their official media partner! As the media partner, we are sharing our podcast-creation resources, like [StreamYard and Veed.io subscriptions](/about/financials), with PyLadiesCon and PyLadiesCon speakers.
+
+At PyPodcats, we've always advocated for the underrepresented Pythonistas in our community, and we're proud to support PyLadiesCon in 2024.
+
+## What is PyLadiesCon?
+
+(PyLadiesCon)(https://conference.pyladies.com/) is the global conference for the [PyLadies](https://pyladies.com) community, and it is **the place to be** this December! It’s all happening online from December 6-8, 2024, and the best part? It’s FREE. This event is all about empowering PyLadies, spreading knowledge, and embracing diversity in the Python world.
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:19]:
```suggestion
[PyLadiesCon](https://conference.pyladies.com/) is the global conference for the [PyLadies](https://pyladies.com) community, and it is **the place to be** this December! It’s all happening online from December 6-8, 2024, and the best part? It’s FREE. This event is all about empowering PyLadies, spreading knowledge, and embracing diversity in the Python world.
```

File: content/english/blog/pypodcats_supports_pyladiescon.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+- Communications Committee
+- Design & Social Media
+- Sponsorship & Finance
+- Tech & Infrastructure
+- Programs Committee
+- Website Committee
+
+Check out the [volunteer page](https://conference.pyladies.com/volunteer/) and get in on the action!
+
```
Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:71]: We hope you will join us at PyLadiesCon!

Comment (Mariatta) [inline content/english/blog/pypodcats_supports_pyladiescon.md:71]:
```suggestion
We hope that you will join us at PyLadiesCon!
```

**Review by Mariatta — APPROVED** [review]

**Review by Cheukting — APPROVED** [review]
Looks good to me too! Side note, maybe we also need a blog post in PyLadiesCon Website about this?

**Pull request merged**